corehq/apps/app_manager/templates/app_manager/v2/partials/form_workflow.html

```diff
@@ -96,25 +94,25 @@ <h4 class="panel-title panel-title-nolink">
             <input type="hidden" name="datums_json" data-bind="value: formLink.serializedDatums"/>
         </div>
         <div class="row">
-            <div class="col-sm-12" data-bind="visible: !formLink.autoLink()">
+            <div class="col-sm-11" data-bind="visible: !formLink.autoLink()">
                 <p>{% trans "This form requires manual linking." %}</p>
             </div>
-            <div class="col-sm-12" data-bind="visible: formLink.autoLink()">
+            <div class="col-sm-11" data-bind="visible: formLink.autoLink()">
                 <p>{% trans "This form doesn't require manual linking." %}</p>
             </div>
-            <div class="col-sm-12" data-bind="visible: formLink.autoLink() && !formLink.manualDatums()">
+            <div class="col-sm-11" data-bind="visible: formLink.autoLink() && !formLink.manualDatums()">
                 <span data-bind="click: enableManualDatums" class="btn btn-default btn-sm">
                     {% trans "Enable manual linking" %}
                 </span>
             </div>
-            <div class="col-sm-12" data-bind="visible: formLink.manualDatums">
+            <div class="col-sm-11" data-bind="visible: formLink.manualDatums">
                 <span data-bind="click: disableManualDatums" class="btn btn-danger btn-sm">
                     {% trans "Disable manual linking" %}
                 </span>
             </div>
         </div>
         <div class="row">
-            <div class="col-sm-12"  data-bind="visible: formLink.showLinkDatums()">
+            <div class="col-sm-11"  data-bind="visible: formLink.showLinkDatums()">
```

Comment:
where's the sm-1 column complementing this? why did you decide to go with sm-12?